xml/System.Numerics/BigInteger.xml

```diff
@@ -1482,11 +1482,15 @@
 ## Remarks  
  This property is a convenience feature that indicates whether a <xref:System.Numerics.BigInteger> value is evenly divisible by two. It is equivalent to the following expression:  
   
-```csharp  
+ [C#]  
```

Comment:
Yes, please, can you send the instructions offline. thank you. Or maybe i will check again my edited version. thank you.